doc/source/ray-air/trainers.rst

```diff
@@ -164,6 +167,30 @@ training via Pytorch DDP.
         :start-after: __hf_trainer_start__
         :end-before: __hf_trainer_end__
 
+Accelerate
+**********
+
+If you prefer a more fine-grained Hugging Face API than what Transformers provides, you can use :class:`AccelerateTrainer <ray.train.huggingface.accelerate.AccelerateTrainer>`
+to run training functions making use of Hugging Face Accelerate. This Trainer is also an extension of  :class:`TorchTrainer <ray.train.torch.TorchTrainer>`, allowing
+you to pass an Accelerate configuration file generated with ``accelerate config`` to be applied on all training workers. This ensures that the worker
```

Comment:
curious, does this functionality (i.e. being able to pass a config file to all training workers) actually has anything to do with TorchTrainer?

Reply:
no, that's the accelerate trainer functionality

Reply:
so why do you start the sentence with ``This Trainer is also an extension of  :class:`TorchTrainer <ray.train.torch.TorchTrainer>, ...``, if the functionality mentioned in the 2nd half has nothing to do with it?

Reply:
Reworded